src/commands/githubActions/endToEndTest/testCheckers.ts

```diff
@@ -116,10 +127,52 @@ async function ensureNoErrorLogs(
 }
 
 /**
- * Test that the validators are attesting after a peri
+ * Test that the validators are attesting after doppelganger protection + 18 minutes max.
  */
-async function attestanceProof(network: Network): Promise<void> {
-  if (network === "mainnet") return;
-  // TODO
-  return;
-}
+export async function attestanceProof(network: Network): Promise<void> {
+  // Wait for doppleganger protection to end, set to 15 min
+  console.log(chalk.grey(`  - Waiting for doppleganger protection to end`));
+  await new Promise(resolve => setTimeout(resolve, 15 * 60 * 1000));
+
+  let response = undefined; // Define response and give it an initial value of undefined
+  let endpoint = "";
+
+  if (network === "mainnet") {
+    // TODO: Add mainnet endpoint
+    // endpoint = `https://beaconcha.in/api/v1/validator/${process.env.VALIDATOR_INDEX}`;
+  } else if (network === "prater") {
+    endpoint = `https://prater.beaconcha.in/api/v1/validator/${process.env.VALIDATOR_INDEX}`;
+  }
+
+  let i = 1;
+  console.log(chalk.grey(`  - Checking if validator is active`));
+
+  while (i <= 9) {
+   await new Promise(resolve => setTimeout(resolve, 2 * 60 * 1000)); // Wait for 2 minutes before each iteration
+    try {
+      response = await got(endpoint, {
+        headers: {
+          Accept: 'application/json'
+        },
+        responseType: 'json'
+      });
+    } catch (err) {
+      throw new Error(`Error while calling beaconcha.in endpoint ${err}`);
```

Comment:
You are not adding valuable information here, either do not add any try catch block or create a beautiful error message that provdes useful information for debuggin such as:
- network
- Exactly endpoint used
- Clients configuration?
- validator index?